### PR TITLE
appveyor fix

### PR DESCRIPTION
**Pull request by ilya-klyuchnikov**

### CODE REVIEW

**Comment by facebook-github-bot-6** [comment]
By analyzing the blame information on this pull request, we identified @mikekap, @inglorion and @ryu2 to be potential reviewers.

File: appveyor.yml [hunk 1]
```diff
@@ -1,20 +1,27 @@
...
-  - set BUCK_EXTRA_JAVA_ARGS=-Djna.nosys=true
-  - python --version
-  - java -version
-  - ant -version
+  - cmd: choco install ant -y -f -i
+  - cmd: refreshenv
+  - cmd: set BUCK_EXTRA_JAVA_ARGS=-Djna.nosys=true
+  - cmd: set GOROOT=
+  - cmd: set PATH=%PATH:C:\go\bin;=%
+  - cmd: set PATH=%PATH:C:\go\bin=%
```
Comment (Coneko) [inline appveyor.yml]: How does this work? I assume it's for removing it from the path, but I can't find any reference for the syntax.

File: src/com/facebook/buck/io/ExecutableFinder.java [hunk 2]
```diff
@@ -169,6 +169,7 @@ private static boolean isExecutable(Path exe) {
 
     String pathEnv = env.get("PATH");
     if (pathEnv != null) {
+      pathEnv = pathEnv.trim();
```
Comment (Coneko) [inline src/com/facebook/buck/io/ExecutableFinder.java]: Why is this necessary?

**Comment by Coneko** [comment]
Appveyor looks green to me.

**Comment by Coneko** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-1** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/1044487142306984/int_phab) to review.

File: src/com/facebook/buck/io/ExecutableFinder.java (hunk 2 above)
Comment (ilya-klyuchnikov) [inline src/com/facebook/buck/io/ExecutableFinder.java]: At appvoyer Path ends with the space (unfortunately). Then a java `Path` object is constructed. It is illegal to have trailing spaces in file names on windows.

File: appveyor.yml (hunk 1 above)
Comment (ilya-klyuchnikov) [inline appveyor.yml]: http://www.dostips.com/DtTipsStringManipulation.php#Snippets.Remove
The subtlety is that `C:\go\bin` may be in the end of the path or in the middle, that is why I have two cases here.
